package.json

```diff
@@ -80,7 +80,7 @@
     "istanbul-middleware": "=0.2.2",
     "jsdoc": "=3.4.3",
     "jsdox": "=0.4.10",
-    "lisk-js": "=0.4.3",
+    "lisk-js": "git://github.com/LiskHQ/lisk-js#optional_data_test",
```

Comment:
Because this PR introduces a new field (i.e. data property) with type 0 transaction, which isn't acceptable by the network until lisk core 1.0.0 is released. So, we can't use a published version of lisk-js. @Tosch110 will probably merge this branch with lisk-js 1.0.0 branch, and then I'll update this line.